### ISSUE TITLE
BNGParser should throw an error if $ sign (fixed flag) is used in 'reaction rules' block

### ISSUE THREAD

**Issue by GoogleCodeExporter** [RuleWorld/bionetgen#65]
```
The use of the $ sign in the 'reaction rules' (but not 'seed species') block is 
scheduled to be deprecated (see Issue 64). In anticipation of this, the 
BNGParser should throw an error if a $ sign is found in a reaction rule 
specification. Examples are in the attached model file.

Note that $ signs in the reaction rules break the CMap generation in RB. Having 
the BNGParser throw errors will thus allow the user to identify the source of 
this problem.
```

Original issue reported on code.google.com by `lh64@cornell.edu` on 30 Nov 2012 at 10:38

Attachments:
- [egfr_net_red.bngl](https://storage.googleapis.com/google-code-attachments/bionetgen/issue-65/comment-0/egfr_net_red.bngl)
